apps/frontend/src/pages/auth/auth.tsx

```diff
@@ -39,18 +53,45 @@ const Auth: React.FC = () => {
 		[dispatch],
 	);
 
+	const handleForgotPasswordSubmit = useCallback(
+		(payload: EmailDto): void => {
+			void dispatch(authActions.requestResetPassword(payload));
+			navigate(AppRoute.SIGN_IN);
+		},
+		[dispatch, navigate],
+	);
+
+	const handleResetPasswordSubmit = useCallback(
+		(payload: ResetPasswordDto): void => {
+			void dispatch(
+				authActions.resetPassword({
+					jwtToken: payload.jwtToken,
+					newPassword: payload.newPassword,
+				}),
+			);
+			navigate(AppRoute.SIGN_IN);
+		},
+		[dispatch, navigate],
+	);
+
 	const getScreen = (screen: string): React.ReactNode => {
-		switch (screen) {
-			case AppRoute.SIGN_IN: {
+		switch (true) {
```

Comment:
Should be `switch (screen)`
For Reset password page, you can add token in query params

Reply:
I already pass the token from query params
<img width="1440" alt="Screenshot 2024-09-01 at 11 34 22" src="https://github.com/user-attachments/assets/c73f6577-8659-4763-af41-a3bedcff4607">
The problem is that i can't match the dynamic segment inside of /reset-password/:id since it wants an EXACT url to match, that is why i made a function that matches incoming pathname (screen) agains a regex that checks for "reset-password"

Reply:
Or do you mean something like `case pathname == "${AppRoute.RESET_PASSWORD}/${token}"`?